dll/shellext/netshell/lang/pt-PT.rc

```diff
@@ -106,13 +113,13 @@ END
 
 IDD_NETWORKDOMAINPAGE DIALOGEX 0, 0, 317, 143
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Instalador do ReactOS"
+CAPTION "Programa de configuração do ReactOS"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Pretende que este computador seja membro de um domínio?\n\
-(Pode obter esta informação do seu administrador de rede.)", IDC_STATIC, 33, 2, 263, 50
-    AUTORADIOBUTTON "Não, este computador fará parte de um grupo de &trabalho.", IDC_SELECT_WORKGROUP, 33, 32, 253, 20, WS_TABSTOP | WS_GROUP
-    AUTORADIOBUTTON "Sim, este computador fará parte de um &domínio.", IDC_SELECT_DOMAIN, 33, 47, 253, 20, WS_TABSTOP
+    LTEXT "Deseja que este computador seja membro de um domínio?\n\
+(Pode obter esta informação no seu administrador de rede.)", IDC_STATIC, 33, 2, 263, 50
+    AUTORADIOBUTTON "Não; fazer este computador membro do seguinte grupo de trabalho.", IDC_SELECT_WORKGROUP, 33, 32, 253, 20, WS_TABSTOP | WS_GROUP
```

Comment:
Why did you strip the accelerator? To be honest, I can't spot a single substantial improvement in this PR, except line 119 Pretende -> Deseja and maybe the compone**n**tes typo-fix.

Reply:
In WinSrv2003 there is no accelerator in this text.

![Grupo](https://github.com/reactos/reactos/assets/49964946/55c23b76-4504-4336-8c53-c95e2b353402)

Reply:
Have you pressed the Alt-key in that dialog? Accels may be configured as being hidden by default.

Reply:
The `Next >` button has an underline, so I think yes.

Reply:
Ah no, nevermind, it's just "g".